### DISCUSSION TITLE
Exclude schema.table from automap_base.prepare auto-mapping

### DISCUSSION THREAD

**Discussion by stig2022** [sqlalchemy/sqlalchemy#11208] [Usage Questions]
While using automap_base.prepare(autoload_with=engine), where the engine points at a specified schema, I'd like to exclude one (or more) particular tables in the target schema from my auto-mapping. 

Is there any way to do this? I've searched around and it appears that there may be a way using reflection_options - however I'm unsure if this is true, and if so, what the correct syntax would be.

Any assistance would be greatly appreciated!

**Comment by CaselIT** [comment]
Hi,

It's documented here https://docs.sqlalchemy.org/en/20/orm/extensions/automap.html#sqlalchemy.ext.automap.AutomapBase.prepare and https://docs.sqlalchemy.org/en/20/core/metadata.html#sqlalchemy.schema.MetaData.reflect
